src/compatTest/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPluginTests.kt

```diff
@@ -541,6 +541,64 @@ class NexusPublishPluginTests {
         assertThat(result.output).contains("SocketTimeoutException")
     }
 
+    @Test
+    fun `uses s01 default URLs for sonatype repos in Groovy DSL`() {
```

Comment:
These two tests are very similar to the two for the old instance. Wouldn't be possible to cover them with `junit-jupiter-params`?

Reply:
Done :+1: